frontend/src/index.css

```diff
@@ -1,9 +1,17 @@
 * {
 	box-sizing: border-box;
+	width: 100%;
+}
+
+#root {
+	max-width: 1200px;
```

Comment:
Depending how this is applied, this could potentially cause unintended horizontal scrolling in the web app.